tests/test-values.yaml

```diff
@@ -497,3 +497,9 @@ forceSchemaArrayModifier:
   - w: "x"
     y:
       - "z"
+
+## Invented parameter to test proper modifier consumption
+## @param modifierWithSquareBrackets [object] Additional square brackets should appeare in description. [More information here](#additional-square-brackets).
```

Comment:
```suggestion
## @param modifierWithSquareBrackets [object] Additional square brackets should appear in description. [More information here](#additional-square-brackets).
```

Reply:
Changed the description completely, should better reflect the purpose of the test.